### ISSUE TITLE
Is there any flag to disable analytics from https://static.cloudflareinsights.com? in docker image

### ISSUE THREAD

**Issue by istiak101** [louislam/uptime-kuma#2954]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hi @louislam,

Is there any docker environment variable to disable this analytics from Cloudflare? I believe you have good intentions to get that analytics but I do not trust any significantly big corporation. This is the same reason I self host things.

`louislam/uptime-kuma   1         8ac0fab1d435   24 hours ago   485MB`

Thanks

### 🐻 Uptime-Kuma Version

1.21.0

### 💻 Operating System and Arch

N/A

### 🌐 Browser

N/A

**Comment by louislam** [comment]
Uptime Kuma do not have this.
What exact place did you find this?

**Comment by rezzorix** [comment]
> If you look into the source code on GitHub or inside the image, you cannot find this code.
> 
> My guess is it is an issue in your side. Your domain name is probably hosted on Cloudflare DNS and enabled auto injecting tracking code. You should check this in your Cloudflare dashboard.

Indeed Cloudflare injects this beacon.min.js - it is not part of the original code of Uptime Kuma.
Some more information on that beacon here: https://developers.cloudflare.com/analytics/faq/web-analytics/

**Comment by istiak101** [comment]
Sorry I couldn't reply to this earlier. I have found what happened at the same time you replied but couldn't reply due to my connectivity. I have many domains hosted with Cloudflare and in only one of them Cloudflare automatically activated web analytics without me doing anything and this was the domain that I hosted Uptime Kuma with. So I was confused why my other domains don't had such script loaded as I was checking them.

Sorry for causing you trouble, indeed you said the truth. As for me I am going to abandon Cloudflare soon and will move to a selfhosted solution that I am currently developing. Thanks for your help. May you receive the same goodness that you did.
